docker/fetch-data/Dockerfile

```diff
@@ -20,4 +28,9 @@ COPY ./domain_to_rank.p /ml/domain_to_rank.p
 COPY ./word_to_ngram.p /ml/word_to_ngram.p
 COPY ./word_to_regex.p /ml/word_to_regex.p
 
+COPY --from=files-src /distilbert-base-uncased.onnx /ml/distilbert-base-uncased.onnx
+
+COPY ./distilled_bert_base_uncased_tokenizer.zip /ml/distilled_bert_base_uncased_tokenizer.zip
+RUN apt-get install unzip && unzip /ml/distilled_bert_base_uncased_tokenizer.zip -d /ml && rm /ml/distilled_bert_base_uncased_tokenizer.zip
```

Comment:
@lperryp I think we can reduce the image size if we do the unzip inside the build container and copy the unzipped directory.